deb/debian/postinst

```diff
@@ -51,9 +49,11 @@ case "$1" in
 cat << EOF
 The percona-release package now contains a percona-release script that can enable additional repositories for our newer products.
 
-For example, to enable the Percona Server for MySQL 8.0 repository use:
+Note: currently there are not any enabled repositories that contain Percona products or distributions. We recommend you enabling Percona Distribions instead of separate product repos as in this case you will get not only database itself but a set of other commponets that will help you working with your database.
```

Comment:
```suggestion
Note: currently there are no repositories that contain Percona products or distributions enabled. We recommend you to enable Percona Distribution repositories instead of individual product repositories, because with the Distribution you will get not only the database itself but also a set of other componets that will help you work with your database.
```